### PR TITLE
Fix few doc

### PR DESCRIPTION
**Pull request by yangyangtiantianlonglong**
##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: ssl/record/rec_layer_s3.c [hunk 1]
```diff
@@ -870,10 +870,7 @@ int do_ssl3_write(SSL *s, int type, const unsigned char *buf,
             rectype = type;
         SSL3_RECORD_set_type(thiswr, rectype);
 
-        /*
-         * Some servers hang if initial client hello is larger than 256 bytes
-         * and record version number > TLS 1.0
-         */
+        /* Some servers hang if record version number > TLS 1.0 */
```
Comment (t8m) [inline ssl/record/rec_layer_s3.c:873]: I would not change this comment. The information is still relevant although we explicitly do not check the > 256 bytes length.

Comment (yangyangtiantianlonglong) [reply ssl/record/rec_layer_s3.c:873]: `initial client hello is larger than 256 bytes and record version number > TLS 1.0` are these two conditions parallel? 
They also paid attention to these two issues  https://github.com/openssl/openssl/issues/9073  https://github.com/openssl/openssl/issues/1689

Comment (yangyangtiantianlonglong) [reply ssl/record/rec_layer_s3.c:873]: Sorry, I misunderstood the possible conditions. and Will remove commit.

**Review by t8m — APPROVED** [review]

**Comment by mspncp** [comment]
Merged to master, thank you for your contribution!

* 4a4f446008 doc: Refactored the example in crypto.pod
* 5272fdca6b doc: Update "SSL/TLS" old documentation description in openssl.pod

**Pull request closed without merging**